docs/contributing.rst

```diff
@@ -301,7 +301,7 @@ testing. For example, the item loaders code is in::
 
 And their unit-tests are in::
 
-    tests/test_loader.py
+    tests / test_loader.py
```

Comment:
I see there are non-Python code examples affected.

I think the best approach may be _not_ to use any of the 2 command-line options, and instead switch to `.. code-block::` syntax all Python code snippets in the documentation. Would you be OK with going that route?

Reply:
I see, may be I missed some files while reviewing. Alright then, I will consider switching to `.. code-block::` syntax for all python code snippets.

Reply:
However, I will need a few days to make the changes in all files, if that's OK. 😅

Reply:
No rush.